src/audio.rs

```diff
@@ -0,0 +1,37 @@
+use bevy::prelude::*;
+
+/// An organizational marker component that should be added to a spawned [`AudioBundle`] if it is in the
+/// general "music" category (ex: global background music, soundtrack, etc).
+///
+/// This can then be used to query for and operate on sounds in that category. For example:
+///
+/// ```
+/// use bevy::prelude::*;
+/// use bevy_quickstart::audio::Music;
+///
+/// fn set_music_volume(sinks: Query<&AudioSink, With<Music>>) {
+///     for sink in &sinks {
```

Comment:
```suggestion
/// fn set_music_volume(sink_query: Query<&AudioSink, With<Music>>) {
///     for sink in &sink_query {
```

Reply:
Context: this is for consistency. Don’t care much about which version we ise, but we habe been consistent with _queue for now.

Reply:
I'm generally in favor of "plural short names for queries" instead of tacking on `_query` to everything. But happy to align for now.